applications/main/archive/helpers/archive_browser.c

```diff
@@ -351,15 +352,34 @@ void archive_add_app_item(ArchiveBrowserView* browser, const char* name) {
     ArchiveFile_t_clear(&item);
 }
 
+static bool archive_get_fap_meta(string_t file_path, string_t fap_name, uint8_t** icon_ptr) {
+    FapLoader* loader = malloc(sizeof(FapLoader));
```

Comment:
App internals should not be available outside. Implement `fap_loader_alloc();` and fap_loader_free();.